base/linalg/cholesky.jl

```diff
@@ -491,6 +491,21 @@ function A_ldiv_B!(C::CholeskyPivoted, B::StridedMatrix)
     end
 end
 
+\(::Cholesky, ::RowVector) = throw(DimensionMismatch("Cannot left-divide matrix by transposed vector"))
+\(::CholeskyPivoted, ::RowVector) = throw(DimensionMismatch("Cannot left-divide matrix by transposed vector"))
+\(::Cholesky{<:Number}, ::RowVector{<:Number}) = throw(DimensionMismatch("Cannot left-divide matrix by transposed vector"))
```

Comment:
Yes - sorry I wasn't clear, I think these (and others) need to be deleted in order to be consistent. For `*`, `/` and `\`, just let rowvector behave like a matrix and let the existing code see if the dimensions mismatch (some methods might be needed to avoid ambiguities - in this case they need to reshape the rowvector into a matrix).